src/patternfly/components/Form/examples/form-text-input-example.hbs

```diff
@@ -1,46 +1,18 @@
 {{#> form}}
   {{#> form-group}}
     {{#> form-label for="textInput1"}}Standard input{{/form-label}}
-    {{#> form-control}}
-      {{#> form-input id="textInput1" type="text" name="inputName1"}}{{/form-input}}
-    {{/form-control}}
+    {{#> form-controls controlType="input" input="true" type="text" name="inputName1" id="textInput1"}}{{/form-controls}}
   {{/form-group}}
   {{#> form-group}}
     {{#> form-label for="textInput2"}}Input with placeholder{{/form-label}}
-    {{#> form-control}}
-      {{#> form-input id="textInput2" type="text" name="inputName2" placeholder="Example2"}}{{/form-input}}
-    {{/form-control}}
+    {{#> form-controls controlType="input" input="true" type="text" name="textInput2" id="textInput2" placeholder="Placeholder input example"}}{{/form-controls}}
   {{/form-group}}
   {{#> form-group}}
     {{#> form-label for="textInput3" required="true"}}Required input{{/form-label}}
-    {{#> form-control}}
-      {{#> form-input id="textInput3" type="text" name="inputName3" required="true"}}{{/form-input}}
-    {{/form-control}}
+    {{#> form-controls controlType="input" input="true" pf-c-form-control--modifier="pf-m-required" type="text" name="textInput3" id="textInput3" required="true"}}{{/form-controls}}
   {{/form-group}}
   {{#> form-group}}
     {{#> form-label for="textInput4" pf-c-form--modifier="pf-m-disabled"}}Disabled input{{/form-label}}
-    {{#> form-control}}
-      {{#> form-input id="textInput4" type="text" name="inputName4" disabled="true"}}{{/form-input}}
-    {{/form-control}}
-  {{/form-group}}
-  {{#> form-group}}
-    {{#> form-label for="textInput5" pf-c-form-control--modifier="pf-m-required"}}Label with tooltip{{/form-label}}
-    <div class="pf-c-tooltip">
-      <i class="fas fa-info-circle"></i>
-    </div>
-    {{#> form-control}}
-      {{#>button btnClass="pf-m-secondary"}}testing{{/button}}
-      {{#> form-input newcontent id="textInput5" type="text" name="inputName5" aria-describedby="helper1" required="true"}}{{/form-input}}
-      {{#>button btnClass="pf-m-secondary"}}testing{{/button}}
-    {{/form-control}}
-    {{#> form-helper-text id="helper1"}}This is helper text{{/form-helper-text}}
-  {{/form-group}}
-  {{#> form-group}}
-    {{#> form-label for="textInput6" pf-c-form--modifier="pf-m-required"}}Required input with tooltip and error{{/form-label}}
-    <a href="#" class="pf-m-right">Test link</a>
-    {{#> form-control}}
-      {{#> form-input pf-c-form--modifier="pf-m-error" id="textInput6" type="text" placeholder="Has error" aria-errormessage="helper2" aria-invalid="true"}}{{/form-input}}
-    {{/ form-control}}
-    {{#> form-helper-text pf-c-form--modifier="pf-m-error" id="helper2"}}This is helper text{{/form-helper-text}}
+    {{#> form-controls controlType="input" input="true" pf-c-form-control--modifier="pf-m-disabled" type="text" name="textInput4" id="textInput4"  disabled="yes" aria-label="disabled text input" placeholder="Disabled input example"}}{{/form-controls}}
```

Comment:
Isn't part of this PR, but the `<input type="text">` form control element doesn't need a self closing `/>`. Should I open a separate issue for that?

Reply:
hmm, I'm interested in hearing what some of the JS devs think about this. Is it invalid to keep the closing `/>` for inputs? It is required when defining html in JSX for react. I would assume it's fine to make this change, since I don't think anything automagical is happening between PF4 core and PF4 react, but it's worth checking.

@dmiller9911 or @dgutride - Do you have any thoughts on this one?

Reply:
ok to make the change if that is what is preferred here.  Nothing automatic.....yet.  I do tend to copy the HTML output from the docs, but it needs some conversions anyways.  Also, It looks like the output HTML in the example does not include the `/>`. I don't look at the handlebars at all.